CHANGELOG.md

```diff
@@ -5,13 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog], and this project adheres to
 [Semantic Versioning].
 
-## [Unreleased]
+## [0.4.0] - 2022-11-08
 
 ### Added
 - [#6](https://github.com/MAAP-Project/gedi-subsetter/issues/6): Allow user to
   specify which BEAMs to subset
 
-## [0.3.0] - 2022-10-31
+## [0.3.0] - 2022-10-31 [YANKED]
```

Comment:
@wildintellect this is part of the [keep a changelog](https://keepachangelog.com/en/1.0.0/) guideline. See "What about yanked releases?" in the FAQ section at that link.

@jjfrench technically, we want 0.3.1, not 0.4.0, based on Semantic Versioning conventions that we're using because this is a bug fix release.

Reply:
@wildintellect this is part of the [keep a changelog](https://keepachangelog.com/en/1.0.0/) guideline. See "What about yanked releases?" in the FAQ section at that link.

Reply:
Ah interesting, I'll leave my debates over language with the author of that guide out of this PR review then.